src/calculateRentalCost.js

```diff
@@ -4,7 +4,7 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
```

Comment:
The function `calculateRentalCost` is currently empty and does not implement the required functionality as per the task description. You need to implement the logic to calculate the rental cost based on the number of days, including discounts for renting 3 or more days, and 7 or more days. Refer to the checklist for using constants instead of literals and structuring your if conditions.